Interactive list

peterdgreat opened the pull request:
- Event Listener added to checkbox to switch between true and false when changed
- Local storage is added to keep user data persistent on the page since users data is stored in an array and updated when the checkbox is clicked

peterdgreat committed:
false/True on checkbox

- a CSS strike added when Todo-List is marked completed

peterdgreat committed:
Local storage added

- On reload page stored data is not lost

BrianSammit commented on src/index.js:
```diff
@@ -2,20 +2,38 @@ import Tasks from './tasks.js';
     const input = document.createElement('input');
     input.type = 'checkbox';
     input.checked = task.completed;
+    if (task.completed === true) {
+      li.classList.add('completed');
+    } else {
+      li.classList.remove('completed');
+    }
     input.addEventListener('change', () => {
       task.completed = input.checked;
       if (task.completed === true) {
         li.classList.add('completed');
+        task.completed = true;
+      } else {
+        li.classList.remove('completed');
+        task.completed = false;
```
- [ ] It would be so much better if you write this action in a method and imported it as a module. Modules help you to develop and maintain the app easier running into minor bugs when you need to change or maintain the app. [Here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/lessons/lesson_es6_modules.md) is a reminder of modules. 

![Capture](https://user-images.githubusercontent.com/64173189/134580100-44a96464-69ce-475b-b3ed-3e2b830a26fd.JPG)

Please fix this minor issue 🛠️ 😄

BrianSammit reviewed (changes requested):
# CHANGES REQUIRED 🔴 🛑
Hello @peterdgreat, great job you are doing so far in this project. 💯
TO HIGHLIGHT:
- Your project is professional 
- Your repo is professional 
- Great job doing the checkbox action

However, there are a few changes to make to move to the next project 🛠️ . Keep working hard 💪 💻 

## Required Changes ♻️
See my inline comment to know what changes are needed. 👇 👀 

Please make these required changes and send it again for another review 👍 💪
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# HAPPY CODING!!! 😄 

![programming-happy-fast-gif](https://user-images.githubusercontent.com/64173189/134580227-bab165e7-55c1-4b65-a4e2-cb8ddc22c222.gif)


**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

peterdgreat committed:
status changes from the tasks.js method

- A method for checkbox status changes created on tasks.js

- after importing, it was called upon the event listener on checkbox changes

wandji20 reviewed (approved):
## Hi @peterdgreat  :raised_hands:

##### You have done exceptionally well, and have met all requirements for this project :+1:

# Status: Approved 👍 :partying_face: 

Your project has been approved, Keep up the good work.

## Congratulations :+1:  :fireworks: 
## Optional
- [ optional ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, etc).

_Go ahead and merge this branch. Also, don't forget to submit the project completion form.
Good luck with your next project :mechanical_arm:  :slightly_smiling_face:_ 

 If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
